docs/reference/r-releases/r-rev185_149.md

```diff
@@ -0,0 +1,109 @@
+# Revision 188/149
+<sub>???, Nov ??, 2024</sub>
+
+Dear community,
+
+We are glad to report that we have just released a new updated version for the Charmed Kafka bundles on the `3/stable` channel, 
+upgrading Charm ZooKeeper revision from 136 to 149, and Charmed Kafka from xxx to 188.  
+
+The new release comes with a number of new features from the charms, from Juju secrets support, OAuth/OIDC authentication support, various improvements in the UI/UX and dependencies upgrades.  
+
+Please reach out should you have any question, comment, feedback or information. You can find us here in [Matrix](https://matrix.to/#/#charmhub-data-platform:ubuntu.com) or also on [Discourse](https://discourse.charmhub.io/).
+
+## Kafka
+
+Here below we provide a summary of the main features and bug fixes related to the Kafka operator.
```

Comment:
Uhm, IMHO this sounds a bit of a repetition from the list below, for which I'm not sure I see the value apart from not having section and subsection next to each other (which I didn't know it wasn't allowed)

Reply:
Well, a summary is a short recollection, like a TL;DR. But given the original list of features and bugfixes is well structured and not that long, it may be not that important to provide a summary. At least let's make the introduction shorter, since it doesn't bring much in its current form. I'm suggesting the following:
`New features and bug fixes in the Charmed Apache Kafka operator:`
Not a blocker.

Reply:
sounds good!